README.md

```diff
@@ -1,24 +1,73 @@
-# README
+# RAILS TEMPLATE 
+This repo is a template to be used for creating new projects.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## System dependencies                                                                                                                                  
+  - Ruby (see .ruby-version)
+  - Rails 7.0+
+  - MySQL 8+  
+  - Redis 6.2+
 
-Things you may want to cover:
+## Template Notes
+After creating your new project, you will need to touch some files to customize the new environment. Those files are:  
 
-* Ruby version
+- config/deploy/* to provide the host names for each deployment target.
+- config/deploy.rb to update the project name & other deployment settings.
+- Run `EDITOR=vim rails credentials:edit` to initialize the Rails encrypted credentials files. See config/credentials.example.yml for some of the environment variables that you should copy to 
+config/credentials.yml.enc
+- config/database.yml to reflect the proper database name and other settings. There is an example in `config/database.example.yml`
+- Look at `config/initializers/site_information.rb` and make changes as needed to the constant values shown there.
+- Perform a 'find all' to look for 'fmaprivacy' and 'fmadata'. Modify those values as needed.
```

Comment:
```suggestion
- Perform a 'find all' to look for 'bl_rails_template' and 'BlRailsTemplate' mentions. Modify those values as needed.
```

Reply:
I already removed mentions of fmadata in a separate story, so this part in README should only include bl_rails_template:
```suggestion
- Perform a 'find all' to look for 'bl_rails_template' mentions. Modify those values as needed.
```